apps/redi-talent-pool/src/services/api/api.tsx

```diff
@@ -309,13 +309,16 @@ export async function fetchAllTpJobListingsUsingFilters({
   const filterFederalStates =
     federalStates?.length !== 0 ? { inq: federalStates } : undefined
 
+  const currentDate = new Date()
+
   return http(
     `${API_URL}/tpJobListings?filter=${JSON.stringify({
       where: {
         // loopbackComputedDoNotSetElsewhere__forAdminSearch__fullName: {
         //   like: 'Carlotta3',
         //   options: 'i',
         // },
+        or: [{ expiresAt: { gt: currentDate } }, { exists: false }],
```

Comment:
I am testing this PR on my local and see neither active nor expired job postings. Not sure if the problem is here, but it seems that fetching job postings returns `undefined`.